packages/calendar-time-utils/src/calendar-time.ts

```diff
@@ -43,6 +43,27 @@ export const changeTime = (
 export const getPreviousDay = (day: MomentInput) =>
   moment(day).subtract(1, 'day').format('YYYY-MM-DD');
 
+export const formatDefaultTime = (time: string, locale: LocaleSpecifier) => {
+  const today = moment();
+  if (moment(time, 'HH:mm', true).isValid()) {
+    const [hour, minute] = time.split(':');
+    today.set({
+      hour: parseInt(hour, 10),
+      minute: parseInt(minute, 10),
+    });
+    return moment(today).locale(locale).format('LT'); // 5:13 PM
+  } else {
+    console.warn(
+      `DataTimeInput: the specified defaultDaySelectionTime '${time}' is not supported. The format should be hh:mm, e.g. 11:10. Using 00:00 as default time.`
+    );
+    today.set({
+      hour: 0,
+      minute: 0,
+    });
+    return moment(today).format('LT'); // 12:00 AM
```

Comment:
I think we should also use the `locale` here so we don't always return '12:00 AM` as there are locales where we should be returning `00:00`.

Reply:
definitely! thanks